src/set_theory/schroeder_bernstein.lean

```diff
@@ -2,12 +2,17 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro
-
-The Schröder-Bernstein theorem, and well ordering of cardinals.
 -/
 import order.fixed_points
 import order.zorn
 
+/-!
+# Schröder-Bernstein theorem, well-ordering of cardinals
+
+This file proves (nonconstructively) the Schröder-Bernstein theorem (see `schroeder_bernstein`), the
```

Comment:
I agree with Eric: this sentence currently carries no information that is not in the title. You should informally state the Schröder-Bernstein theorem. Even recalling the definition of "well-ordering" would be nice. And I would also remove the "nonconstructively" since  1) we don't care and 2) there cannot be any constructive proof anyway. Your phrasing sounds like there could be a constructive proof but we didn't bother.

Reply:
I got confused between EM and AC. And indeed after inspection it doesn't seem like `schroeder_bernstein` uses AC. Am I right? Is there a systematic way to see?